### PR TITLE
Update index.md with IoT Embedded compatibility information

### PR DESCRIPTION
**Pull request by DavidShoe**
Updates to the platforms section to include IoT Enterprise versions supported.

### CODE REVIEW

**Review by jasonstephen15 — APPROVED** [review]

**Comment by jasonstephen15** [comment]
Approved

File: microsoft-edge/webview2/index.md [hunk 1]
```diff
@@ -167,6 +167,14 @@ You may run WebView2 apps on the following versions of Windows.
 > [!IMPORTANT]
 > \*\* WebView2 support for Windows 7 and Windows Server 2008 R2 has the same support cycle as Microsoft Edge.  For more information, navigate to [Microsoft Edge supported Operating Systems][DeployedgeMicrosoftEdgeSupportedOS].  
 
+WebView2 also works in the following Windows Embedded releases
+
+*   Windows IoT Enterprise LTSC x32
+*   Windows IoT Enterprise LTSC x64
+*   Windows IoT Enterprise 21h1 x64
+
+Windows IoT Enterprise on Arm64 is currently under investigation.
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:176]: @jasonstephen15  - may we drop this one until we know for sure?

File: microsoft-edge/webview2/index.md [hunk 2]
```diff
@@ -153,7 +153,7 @@ A General Availability \(GA\) or Preview version is available on the following p
 *   .NET 5  
 *   [WinUI 3.0][UwpToolkitsWinui3] \(Preview\)  
     
-You may run WebView2 apps on the following versions of Windows.  
+You may run WebView2 apps on the following versions of Windows desktop.
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:156]:
```suggestion
You may run WebView2 apps on the following versions of Windows.  
```

Comment (Reezaali) [reply microsoft-edge/webview2/index.md:156]: Reverting to previous text to use the product name.

File: microsoft-edge/webview2/index.md [hunk 3]
```diff
@@ -167,6 +167,14 @@ You may run WebView2 apps on the following versions of Windows.
 > [!IMPORTANT]
 > \*\* WebView2 support for Windows 7 and Windows Server 2008 R2 has the same support cycle as Microsoft Edge.  For more information, navigate to [Microsoft Edge supported Operating Systems][DeployedgeMicrosoftEdgeSupportedOS].  
 
+WebView2 also works in the following Windows Embedded releases
+
+*   Windows IoT Enterprise LTSC x32
+*   Windows IoT Enterprise LTSC x64
+*   Windows IoT Enterprise 21h1 x64
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:174]: Can we replace with the product names?

File: microsoft-edge/webview2/index.md [hunk 4]
```diff
@@ -167,6 +167,14 @@ You may run WebView2 apps on the following versions of Windows.
 > [!IMPORTANT]
 > \*\* WebView2 support for Windows 7 and Windows Server 2008 R2 has the same support cycle as Microsoft Edge.  For more information, navigate to [Microsoft Edge supported Operating Systems][DeployedgeMicrosoftEdgeSupportedOS].  
 
+WebView2 also works in the following Windows Embedded releases
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:170]:
```suggestion
WebView2 also works on the following Windows Embedded releases.
```

**Review by Reezaali — CHANGES_REQUESTED** [review]

File: microsoft-edge/webview2/index.md (hunk 1 above)
Comment (DavidShoe) [reply microsoft-edge/webview2/index.md:176]: We actually wanted that in there to show we are looking at it (it should work but we need to verify).

File: microsoft-edge/webview2/index.md (hunk 3 above)
Comment (DavidShoe) [reply microsoft-edge/webview2/index.md:174]: Those are the product names as far as I know.

Comment (DavidShoe) [reply microsoft-edge/webview2/index.md:174]: https://docs.microsoft.com/en-us/windows/iot/iot-enterprise/getting_started

**Comment by jasonstephen15** [comment]
Looks like this doc is ready to merge!

**Review by Reezaali — APPROVED** [review]

**Pull request merged**